app/src/main/java/org/mozilla/fenix/home/PocketUpdatesMiddleware.kt

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import mozilla.components.lib.state.Action
+import mozilla.components.lib.state.Middleware
+import mozilla.components.lib.state.MiddlewareContext
+import mozilla.components.service.pocket.PocketStoriesService
+
+/**
+ * [HomeFragmentStore] middleware reacting in response to Pocket related [Action]s.
+ */
+class PocketUpdatesMiddleware(
+    private val coroutineScope: CoroutineScope,
+    private val pocketStoriesService: PocketStoriesService
+) : Middleware<HomeFragmentState, HomeFragmentAction> {
+    override fun invoke(
+        context: MiddlewareContext<HomeFragmentState, HomeFragmentAction>,
+        next: (HomeFragmentAction) -> Unit,
+        action: HomeFragmentAction
+    ) {
+
```

Comment:
nit: no new line necessary

Reply:
That change is not part of this PR. Will update in https://github.com/mozilla-mobile/fenix/pull/21277.